stand_mean: 3.0449262838039544,-81.17859671640707,34.03485779741146
stoop_mean: 0.5814177419029695,-35.79083503567389,37.662095404848
